common/src/java/com/zimbra/common/account/ZAttrProvisioning.java

```diff
@@ -5720,6 +5712,14 @@ public static TwoFactorAuthSecretEncoding fromString(String s) throws ServiceExc
     @ZAttr(id=3101)
     public static final String A_zimbraDocumentRecentlyViewedEnabled = "zimbraDocumentRecentlyViewedEnabled";
 
+    /**
```

Comment:
changing attrs position here seems to have caused unnecessary changes in ZAttrConfig.java  when ran `ant generate-getters`

Reply:
This was auto generated.